Week-2/index2-1.html

```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>

```

Comment:
donot push week1 and week 2 codes together, this time i am reviewing it, from next time you should take care

Reply:
ok mam